alkira/resource_alkira_internet_applications.go

```diff
@@ -162,6 +162,11 @@ func resourceAlkiraInternetApplication() *schema.Resource {
 				},
 				Optional: true,
 			},
+			"ilb_credential_id": {
+				Description: "AWS Credential ID of the ILB Target.  This can only be used if the associated connector is an AWS_VPC",
```

Comment:
What exactly is an ILB? I don't think AWS has this concept at all, so not sure what is the credential for exactly.